app/model/family.py

```diff
@@ -24,25 +24,34 @@ class FamilyReadDTO(BaseModel):
 
 
 class FamilyWriteDTO(BaseModel):
-    """A JSON representation of a family for writing."""
+    """
+    A JSON representation of a family for writing.
+
+    Note:
+     - import_id is given from the request
+     - organisation is immutable
+    """
 
-    # import_id: not included as this is in the request path
     title: str
     summary: str
     geography: str
     category: str
     metadata: Json
-    # organisation: not included as once created is immutable
 
 
 class FamilyCreateDTO(BaseModel):
-    """A JSON representation of a family for creating."""
+    """
+    A JSON representation of a family for creating.
+
+    Note:
+     - import_id is auto generated
+     - slug is auto generated
+     - organisation comes from the user's organisation
+    """
 
     # import_id: not included as generated
```

Comment:
Can we remove this line as we've now added to the docstring that the import_id is auto generated?